u2flib_server/attestation/resolvers.py

```diff
@@ -61,17 +66,35 @@ def add_metadata(self, metadata):
 
     def _index(self, metadata):
         for cert_pem in metadata.trustedCertificates:
-            cert_der = ''.join(cert_pem.splitlines()[1:-1]).decode('base64')
-            cert = X509.load_cert_der_string(cert_der)
-            subject = cert.get_subject().as_text()
+            cert = x509.load_pem_x509_certificate(cert_pem, default_backend())
+            subject = cert.subject.get_attributes_for_oid(NameOID.COMMON_NAME)[0].value
             if subject not in self._certs:
                 self._certs[subject] = []
             self._certs[subject].append(cert)
             self._metadata[cert] = metadata
 
+    def _verify_cert(self, cert, key):
+        cert_signature = cert.signature
+        cert_bytes = cert.tbs_certificate_bytes
+
+        verifier = key.verifier(
```

Comment:
This assumes the issuer key is RSA using PKCS#1 padding, which might not always be the case.